scene/resources/mesh.cpp

```diff
@@ -1565,9 +1565,9 @@ Error ArrayMesh::lightmap_unwrap_cached(int *&r_cache_data, unsigned int &r_cach
 
 	if (!r_used_cache) {
 		//free stuff
-		::free(gen_vertices);
-		::free(gen_indices);
-		::free(gen_uvs);
+		::memfree(gen_vertices);
+		::memfree(gen_indices);
+		::memfree(gen_uvs);
```

Comment:
The `::` can be removed, they were used to disambiguate `free()`, but there's no ambiguity with `memfree()`.

Reply:
Fixed